src/i18n.js

```diff
@@ -62,7 +62,33 @@ const messages = {
     'navbar.about': 'About',
     'navbar.docs': 'Docs',
     'navbar.faq': 'FAQ',
-    'footer.copyright': '© 2023 EOS Network Foundation all rights reserved.'
+    'footer.copyright': '© 2023 EOS Network Foundation all rights reserved.',
+
+    'app.walletconnect.failedswitch.title': 'Missing Network Configuration in Wallet',
+    'app.walletconnect.failedswitch.content': `<p>We've detected that the wallet you are attempting to use has not been configured to support EOS EVM yet. Please use the following network information to add the network or visit our <a href="https://docs.eosnetwork.com/evm/quick-start/introduction" >documentation</a> page for more details.</p>
```

Comment:
Please update to open in new tab and use the following link for EN "https://docs.eosnetwork.com/evm/quick-start/setup-metamask"

Reply:
done